modules/ROOT/partials/configuration/exportword.adoc

```diff
@@ -0,0 +1,85 @@
+[[exportword-service-url]]
+== `exportword_service_url`
+
+By default, the Word export feature uses the {productname} Cloud Services HTML to DOCX converter service to generate the Word files. You can use this option to provide the URL to an on-premises converter.
```

Comment:
Same as the other doc converter plugins, the `exportword_service_url` is required for this plugin to work and it has no default value.

Reply:
Updated in latest commit.